test/e2e/commatrix_suite_test.go

```diff
@@ -215,6 +344,34 @@ var _ = Describe("commatrix", func() {
 	})
 })
 
+// getClusterVersion return cluster's Y stream version.
+func getClusterVersion(cs *client.ClientSet) (string, error) {
```

Comment:
can u put this in utils?
i will use it on my task also
i think ur task will be merge before mine so can u do that?